packages/neos-ui-editors/src/Editors/UriPathSegment/index.js

```diff
@@ -75,6 +75,7 @@ export default class UriPathSegment extends PureComponent {
                         id="neos-UriPathSegmentEditor-sync"
                         size="regular"
                         icon="sync"
+                        disabled={finalOptions.disabled || finalOptions.readonly}
```

Comment:
Ha, that was easy. And we love easy, don't we! ;)

I didn't test this yet, but I'm wondering: Would it make sense to hide the icon completely if the editor is read only?
Not sure about it and this fix would definitely improve the situation already of course!

Reply:
We can also hide it.

Reply:
I'm fine with both.

Reply:
Me too. That was just a brainwave, both make sense as long as it's easy to tell that the icon/button is disabled. Is that the case? (can't test it right now)

Reply:
Removing the button is also easy ... 
Question is remove button as well for disabled as it is more consitent?

Reply:
> remove button as well for disabled as it is more consitent?

Yes, with "read only" above I meant all the cases where the button doesn't have an effect